frontera/contrib/backends/cassandra/revisiting.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can I ask you to review this module and make a little refactoring to make classes like RevistingQueue or Backend reusable? Copy&Paste is fast but not harder to maintain in long-term prospective.